locale/ca/get-involved/index.md

```diff
@@ -12,20 +12,21 @@ contribuir de la manera que puguin. Si vols [reportar un error](https://github.c
 
 ## Discussió de la comunitat
 
-- La [llista d'errors en ](https://github.com/nodejs/node/issues) és el lloc per discutir les característiques del nucli de Node.js.
+- La [llista d'incidències de Github](https://github.com/nodejs/node/issues) és el lloc per discutir les característiques del nucli de Node.js.
 - Per xatejar en temps real sobre el desenvolupament de Node vagi a `irc.freenode.net` al canal `#node.js` fent servir un [client d'IRC](http://es.wikipedia.org/wiki/Anexo:Clientes_IRC) o connecti's amb el seu navegador al canal usant [WebChat de freenode](http://webchat.freenode.net/?channels=node.js).
-- El compte de Twitter oficial de Node.js és  [nodejs](https://twitter.com/nodejs).
-- [Node Weekly](http://nodeweekly.com) és una llista de correu que recopila els últims esdeveniments i notícies al voltant de la comunitat de Node.js.
-- [Planet Node](http://planetnodejs.com) és un agregador de blogs de desenvolupadors de Node.
+- El compte de Twitter oficial de Node.js és [nodejs](https://twitter.com/nodejs).
+- [Node.js Everywhere](https://newsletter.nodejs.org) és el Butlletí oficial mensual de Node.js.
+- [Node.js Collection](https://medium.com/the-node-js-collection) és una col·lecció de contingut comissat per la comunitat a Medium.
 - [NodeUp](http://nodeup.com) és un podcast cobrint les últimes notícies en la comunitat de Node.
 - La [Community Committee](https://github.com/nodejs/community-committee) és una comissió de primer nivell a la Fundació Node.js centrada en els esforços que afronta la comunitat.
 
-
 ## Aprenentatge
 
 - La [Documentació oficial de l'API](/api) detalla l'API de Node.
 - [NodeSchool.io](http://nodeschool.io) li ensenyarà conceptes de Node.js de forma interactiva mitjançant jocs utilitzant la línia de comandes.
 - L'[etiqueta de Node.js en Stack Overflow](http://stackoverflow.com/questions/tagged/node.js) col·lecciona nova informació cada dia.
+- L'[etiqueta de Node.js en la DEV Community](https://dev.to/t/node) és un lloc on compartir projectes de Node.js, articles i tutorials, així com iniciar debats i demanar realimentació sobre temes relacionats amb Node.js. Els desenvolupadors de tots els nivells d'experiència són benvinguts a participar.
+- [Nodeiflux](https://discordapp.com/invite/vUsrbjd) és una comunitat amigable de desenvolupadors de backend amb Node que es recolzen mútuament a Discord.
```

Comment:
`de backend amb Node.js`

I think we should use always the same name for Node.js :)

Reply:
I just translate from the original document in english... where is approved/published just with 'Node'
I am agree with you and consistency using Node.js. 

More opinions?